<launch>
  
  <node pkg="joy" type="joy_node" name="joy_node">
  	<param name="dev" value="/dev/input/js1" />
  </node>
  <node pkg="robucar" type="robucar_joy.py"  name="robucar_joy" output="screen" />
  <node pkg="robucar" type="robucar_control2.py" name="robucar_ctrl" output="screen" />
  <node pkg="robucar" type="robucar_monitor.py"  name="robucar_mon"  output="screen" />

</launch>
